cypress.config.ts

```diff
@@ -1,9 +1,7 @@
-import { defineConfig } from "cypress";
+import { defineConfig } from 'cypress';
 
 export default defineConfig({
   e2e: {
-    setupNodeEvents(on, config) {
-      // implement node event listeners here
-    },
+    baseUrl: 'http://localhost:3000',
```

Comment:
currently the build is broken, but also the tests, it is not running as expected the test action.

Reply:
[3867448](https://github.com/defi-wonderland/web3-react-boilerplate/pull/8/commits/3867448d15ac4b9e4b11d341e62272b4db9855c9)